### PR TITLE
Add parameter file path as global attribute to netCDF output

### LINKED ISSUE #64: NetCDF output should record which parameter file was used
ITS_LIVE requested that the parameter file version be recorded in the netCDF file. 

However, parameter file can now be passed in on the CLI, and there are likely going to be different files for different research applications. 

So, we should record the full path to the file used. 

Unfortunately, we subprocess out to vendored script, so the vendored scripts should be updated upstream to (preferably) allow import and direct calling of script workflow or (alternatively) take parameter file as an input parameter.

### PR DESCRIPTION
**Pull request by jhkennedy**
fixes #64 

In the netCDF global attributes, this will look like:

```
// global attributes:
		...
		:autoRIFT_software_version = "1.2.0" ;
		:autoRIFT_parameter_file = "http://its-live-data.jpl.nasa.gov.s3.amazonaws.com/autorift_parameters/v001/autorift_parameters.shp" ;
		...
````

### CODE REVIEW

File: hyp3_autorift/netcdf_output.py [hunk 1]
```diff
@@ -189,6 +189,7 @@ def netCDF_packaging(VX, VY, DX, DY, INTERPMASK, CHIPSIZEX, CHIPSIZEY, SSM, SSM1
     nc_outfile.setncattr('date_created', datetime.datetime.now().strftime("%d-%b-%Y %H:%M:%S"))
     nc_outfile.setncattr('title', title)
     nc_outfile.setncattr('autoRIFT_software_version', IMG_INFO_DICT["autoRIFT_software_version"])
+    nc_outfile.setncattr('autoRIFT_parameter_file', parameter_file.replace('/vsicurl/', ''))
```
Comment (asjohnston-asf) [inline hyp3_autorift/netcdf_output.py:192]: I'd consider having the calling function be responsible for stripping `/vsicurl/`, but I see this function is called at least six times in the other vendor scripts, so I'll concede that this way is simpler.  Not blocking.

**Comment by asjohnston-asf** [comment]
Should `process()` also pass `parameter_file` to testautoRIFT_ISCE.generateAutoriftProduct()` at https://github.com/ASFHyP3/hyp3-autorift/blob/8ee32ef01d666f9927700a010b01f31663d0418b/hyp3_autorift/process.py#L233 ?

**Comment by jhkennedy** [comment]
> Should `process()` also pass `parameter_file` to testautoRIFT_ISCE.generateAutoriftProduct()` here?
> 
> https://github.com/ASFHyP3/hyp3-autorift/blob/8ee32ef01d666f9927700a010b01f31663d0418b/hyp3_autorift/process.py#L233

Oh good catch!

File: hyp3_autorift/netcdf_output.py (hunk 1 above)
Comment (jhkennedy) [reply hyp3_autorift/netcdf_output.py:192]: Oh yeah, way better to do this in `process.py`

**Review by asjohnston-asf — APPROVED** [review]

**Pull request merged**